iOS/IosVideo.cs

```diff
@@ -223,7 +229,7 @@ void OnReady()
                 else
                     Player.Play();
             }
-
+            View.Duration = ((int)Player.CurrentItem.Duration.Seconds).Seconds();
```

Comment:
View.Duration = Player.CurrentItem.Duration; 

If Player.CurrentItem.Duration an instance of TimeSpan?